src/main/java/org/betonquest/betonquest/quest/event/conversation/ConversationEventFactory.java

```diff
@@ -60,11 +60,37 @@ public Event parseEvent(final Instruction instruction) throws InstructionParseEx
         } catch (final ObjectNotFoundException e) {
             throw new InstructionParseException(e.getMessage(), e);
         }
+        final String startingOption = getStartOption(instruction, conversationID);
         return new PrimaryServerThreadEvent(
                 new OnlineProfileRequiredEvent(
-                        loggerFactory.create(ConversationEventFactory.class), new ConversationEvent(loggerFactory, conversationID), instruction.getPackage()
+                        loggerFactory.create(ConversationEventFactory.class), new ConversationEvent(loggerFactory, conversationID, startingOption), instruction.getPackage()
                 ), server, scheduler, plugin
         );
     }
+
+    /**
+     * Gets an optional start option for the conversation.
+     * This can't verify if the option is actually valid
+     * because the conversation data is not loaded at event creation.
+     *
+     * @param instruction    to get option name from
+     * @param conversationID to get option from
+     * @return null if no option argument is given, otherwise the option name
+     * @throws InstructionParseException if no NPC option with the given name is present
+     */
+    private String getStartOption(final Instruction instruction, final ConversationID conversationID) throws InstructionParseException {
+        final String targetOptionName = instruction.getOptional("option");
+        if (targetOptionName == null) {
+            return null;
+        }
+
+        final String optionPath = "conversations." + conversationID.getBaseID() + ".NPC_options." + targetOptionName;
```

Comment:
```suggestion

        // We need to manually check the existence of the starting option because the conversation is not loaded yet.
        final String optionPath = "conversations." + conversationID.getBaseID() + ".NPC_options." + targetOptionName;
```